zproject/backends.py

```diff
@@ -1354,6 +1358,12 @@ class GitLabAuthBackend(SocialAuthMixin, GitLabOAuth2):
     # we just use the primary email address, which is always verified.
     # (No code is required to do so, as that's the default behavior).
 
+@external_auth_method
+class AppleAuthBackend(SocialAuthMixin, AppleIdAuth):
+    sort_order = 10
+    name = "apple"
+    auth_backend_name = "Apple"
```

Comment:
we fill want `full_name_validated = True` on this backend I think, for a smoother sign up experience

Reply:
Okay! added that. also it wouldn't be any problem as the user can change the name later in settings.